public/js/nav.js

```diff
@@ -0,0 +1,16 @@
+var navbar = document.getElementsByClassName('nav-bar')[0]
+
+// trigger this function every time the user scrolls
+window.onscroll = function (event) {
+  var scroll = window.pageYOffset
+
+  if (scroll > 30) {
+    navbar
+      .querySelectorAll('.navbar')[0]
+      .setAttribute('style', 'background-color:#212529 !important;')
```

Comment:
two ideas
* Move this background color to a CSS class and on trigger, add/remove that class to the navbar.
* use jQuery to accomplish this. I think it should look something like this: (haven't tested it)
```suggestion
    $('.navbar').addClass('someclassname')
```

Reply:
Done